SU-SWS/saa_victoria_subtheme
# Adapt 443  fat footer

# FINALLY READY!

# Summary
- Adding fat footer structure and styles, must merge AFTER pr #35

# Review By (Date)
- 8/13 so we have time to review and do design tweaks from Michelle before 8/17 and 8/18

# Urgency
- HIGH

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Footer should be present and styled according to figma:  https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=192%3A626
4. Check that all the footer links are working

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- JIRA ticket - ADAPT-443
-  must merge AFTER pr #35 - ALREADY DONE


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
